vendor/github.com/moby/moby/api/LICENSE

```diff
@@ -1,191 +0,0 @@
-
```

Comment:
Also opened https://github.com/moby/moby/pull/51671 with an alternative approach for the LICENSE files to reduce code-churn.